sechub-web-ui/package.json

```diff
@@ -5,10 +5,12 @@
     "dev": "vite",
     "build": "vue-tsc --noEmit && vite build",
     "preview": "vite preview",
-    "lint": "eslint . --fix --ignore-path .gitignore"
+    "lint": "eslint . --fix --ignore-path .gitignore",
+    "codegen": "rm -rf src/generated-sources/openapi; openapi-generator-cli generate -i ../sechub-openapi-java/src/main/resources/openapi.yaml -o src/generated-sources/openapi -g typescript-fetch --additional-properties=supportsES6=true,npmVersion=10.8.3"
```

Comment:
I don't think that `codegen` is a good name for this operation
It is to generic

- `openapi-client-generate`
- `generate-api-client`
- ...

would be better imo